docs/getting_started.md

```diff
@@ -23,6 +23,12 @@ On GKE, you will need grant your account the ability to create new cluster roles
 kubectl create clusterrolebinding YOURNAME-cluster-admin-binding --clusterrole=cluster-admin --user=YOUREMAIL@gmail.com
 ```
 
+!!! note
+    If you not interested in UI, SSO, multi-cluster management and just want to pull changes into the cluster then deploy Argo CD using
```

Comment:
Minor: It's missing an "are" after "If you" \</nitpick\>

Reply:
thank you! fixed